Documentation/ApiOverview/Fluid/Introduction.rst

```diff
@@ -10,8 +10,8 @@ Fluid is TYPO3’s default rendering engine but can also be used in a standalone
 The Fluid source code is being developed as an independent project outside of the TYPO3 Core.
 
 Fluid is based on XML.
-You can use HTML markup in Fluid, but you can do much more with Fluid, such as use conditions,
-variables or custom ViewHelpers which are PHP components.
+You can use HTML markup in Fluid. Fluid ViewHelpers offer you different controls, conditions
```

Comment:
What do you mean with "controls" here

Reply:
for each loops

Reply:
I don't agree with changing this wording. I feel like we lose some meaning if we delete "...but you can do much more with Fluid". However, I'm not familiar with this subject matter. If we really need to introduce the concept of _controls_, I'd be inclined to do it by adding another sentence rather than changing what is here.

Reply:
I made another proposal